include/nil/crypto3/zk/snark/systems/plonk/placeholder/detail/transcript_initialization_context.hpp

```diff
@@ -107,27 +107,23 @@ namespace nil {
 
                         // Marshall the initialization context and push it to the transcript.
                         using Endianness = nil::marshalling::option::big_endian;
-                        using TTypeBase = nil::marshalling::field_type<Endianness>;
-                        using value_marshalling_type = nil::crypto3::marshalling::types::transcript_initialization_context<
-                            TTypeBase, nil::crypto3::zk::snark::detail::transcript_initialization_context<PlaceholderParamsType>>;
                         auto filled_context = nil::crypto3::marshalling::types::fill_transcript_initialization_context<
                             Endianness, nil::crypto3::zk::snark::detail::transcript_initialization_context<PlaceholderParamsType>>(context);
 
                         std::vector<std::uint8_t> cv(filled_context.length(), 0x00);
                         auto write_iter = cv.begin();
-                        nil::marshalling::status_type status = filled_context.write(write_iter, cv.size());

```

Comment:
Let's not delete this statuses, but add a BOOST_ASSERT on them, check that they are == success everywhere.